pkgs/top-level/variants.nix

```diff
@@ -90,6 +90,22 @@ self: super: {
     else
       throw "Musl libc only supports 64-bit Linux systems.";
 
+  pkgsUutils =
+    if stdenv.hostPlatform.isLinux && stdenv.buildPlatform.is64bit then
+      nixpkgsFun {
+        overlays = [
+          (self': super': {
+            pkgsUutils = super';
+          })
+        ]
+        ++ overlays;
+        ${if stdenv.hostPlatform == stdenv.buildPlatform then "localSystem" else "crossSystem"} = {
+          config = lib.systems.parse.tripleFromSystem (makeMuslParsedPlatform stdenv.hostPlatform.parsed);
```

Comment:
musl?

Reply:
I copy pasted this from musl. It might be unecassary or I forgot to change something. I did this by now a few years ago and probably forgot most things about it already.